extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -93,7 +110,9 @@ export class PublishDatabaseDialog {
 	private initializePublishTab(): void {
 		this.publishTab.registerContent(async view => {
 			// TODO : enable using this when data source creation is enabled
+			const flexRadioButtonsModel = this.createPublishTypeRadioButtons(view);
 			this.createRadioButtons(view);
```

Comment:
move `createRadioButtons` up keep it with the TODO comment for whenever data sources gets added back